sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -19,6 +21,127 @@
 from ...utils._encode import _unique
 
 
+def get_categorical_colors_and_mapping(value_list, palette="tab10"):
+    """
+    Derives a list of color and color mapping from the passed value list.
+
+    Parameters
+    ----------
+    value_list : array-like of shape (n_samples,) of str
+        Names of the categorical values by which the ICE lines are colored
+    palette: str
+        name of the matplotlib color palette
```

Comment:
default wert mit angeben

Reply:
ich meinte in der beschreibung, also `name of the matplotlib color palette (default: "tab10")` - oder du setzt es gleich auf None weil du da unten ja auch nochmal checkst ob es None ist und dann setzt, das wäre vllt noch besser (gleiches dann auch bei der funktion unten)